.bazelrc

```diff
@@ -1 +1,4 @@
+# Enable Bzlmod for every Bazel command
+common --enable_bzlmod
```

Comment:
You didn't remove `WORKSPACE`, so if I understand correctly bzlmod-enabled builds will use these new files and non-bzlmod builds will use the old files. Seems we need to add another build to the CI that disables bzlmod.